src/Interpreters/MutationsInterpreter.cpp

```diff
@@ -409,12 +409,13 @@ MutationsInterpreter::MutationsInterpreter(
     , available_columns(std::move(available_columns_))
     , settings(std::move(settings_))
     , select_limits(SelectQueryOptions().analyze(!settings.can_execute).ignoreLimits())
+    , logger(getLogger("MutationsInterpreter"))
```

Comment:
Can we forward storage name here `storage.getStorageID().getFullTableName()`. x10 debugging speed.